<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataRetrieval</class>
 <widget class="QMainWindow" name="DataRetrieval">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataRetrieval</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>20</y>
      <width>801</width>
      <height>334</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Search For Contact</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Feild Name to Search </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="feildName"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Keyword</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="keyword"/>
       </item>
       <item>
        <widget class="QPushButton" name="submit">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Contact Number, VID, Contact Status, Name, Age</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="search"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>All Contact Listings</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Contact Number, VID, Contact Status, Name, Age</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="allContacts"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
